### PR TITLE
Handling user management with keycloak

### LINKED ISSUE #12: Support setting institution for keycloak-based deployment
Within keycloak-based deployment users are managed by keycloak.
As admin user of Record Manager, I would like to be able to:
- set up users in keycloak
- add newly created users to institutions

A/C:
- It is possible to add new user in Keycloak UI and then attach to the user existing institution
```[tasklist]
### Tasks
- [ ] https://github.com/kbss-cvut/record-manager/issues/33
```

### LINKED ISSUE #49: Fix new buttons PR
https://github.com/kbss-cvut/record-manager-ui/pull/22 should be fixed by making tests work and resolves the following issues (Preferably with creating new PR):

- [x] #12
- [x] #46 
- [x] https://github.com/kbss-cvut/record-manager-ui/pull/40#discussion_r1426440894
- [x] https://github.com/kbss-cvut/record-manager-ui/pull/40#discussion_r1425974623
- [x] https://github.com/kbss-cvut/record-manager-ui/pull/40#discussion_r1425973315
- [x] #48

### PR DESCRIPTION
**Pull request by LaChope**
Resolves #49 

- c0fe1368f5cb11269b3f56863ba910e878294afc resolves:
    - #12
    - https://github.com/kbss-cvut/record-manager-ui/pull/40#discussion_r1426440894 
    - https://github.com/kbss-cvut/record-manager-ui/pull/40#discussion_r1425974623
 
- [a80832d](https://github.com/kbss-cvut/record-manager-ui/pull/50/commits/a80832da61c43d4b8b797f935a6a8ff09752a9fb) resolves: 
    - #46

### CODE REVIEW

File: js/components/user/User.js [hunk 1]
```diff
@@ -89,33 +89,23 @@ class User extends React.Component {
...
             return <Button style={{margin: '0 0.3em 0 0'}} variant='primary' size='sm' ref='submit'
-                           onClick={handlers.onPasswordChange}>
-                {this.i18n('user.password-change')}
-            </Button>;
+                           onClick={handlers.onPasswordChange}>{this.i18n('user.password-change')}</Button>;
         }
     }
 
-    _externalEditUserButton() {
+    _redirectToKeycloak() {
```
Comment (blcham) [inline js/components/user/User.js:102]: Please rename, see above.

File: js/components/user/User.js [hunk 2]
```diff
@@ -89,33 +89,23 @@ class User extends React.Component {
         });
     };
 
-    _passwordChangeButton() {
+    _passwordChange() {
```
[the remark this answers is not in the document]
Comment (LaChope) [reply js/components/user/User.js]: Done

File: js/components/user/User.js (hunk 1 above)
Comment (LaChope) [reply js/components/user/User.js]: Done

**Comment by LaChope** [comment]
@blcham Can be closed in favor of #51 for cleaner history purposes

**Comment by blcham** [comment]
Closing as suggested.

**Pull request closed without merging**